### ISSUE TITLE
Change <h1> construction to use visible banner title

### ISSUE THREAD

**Issue by jvine** [sul-dlss/earthworks#690]
There's an oddity with the `<h1>` that seems to be a Blacklight thing - it's also present in SearchWorks.

1. The visible application title (and home link) in the banner is not actually a heading.
2. There is a hidden `sr-only` `<h1>` further down in the DOM, which in some cases ends up in a bad location.

### Home page with the hidden `<h1>` exposed
<img width="500" alt="Screen Shot 2020-07-01 at 5 35 09 PM" src="https://user-images.githubusercontent.com/6945691/86303797-c1a32c00-bbc1-11ea-9f14-a6efa30a6227.png">

### Show page with the hidden `<h1>` exposed
<img width="500" alt="Screen Shot 2020-07-01 at 5 37 32 PM" src="https://user-images.githubusercontent.com/6945691/86303807-c831a380-bbc1-11ea-97b3-c1335185d701.png">

### Search results with several hidden headings exposed
This one is really broken - resulting structure is:
- `<h1>` Search
   - `<h2>` Search Constraints
- `<h1>` EarthWorks Geospatial Catalog
   - `<h2>` Search Results

<img width="500" alt="Screen Shot 2020-07-01 at 5 43 51 PM" src="https://user-images.githubusercontent.com/6945691/86304535-f912d800-bbc3-11ea-8401-fc877f4fc059.png">

# Solution

- [ ] Make the visible title ALSO the `<h1>`, so that it is physically and logically at the top of the page. 

   If the title needs additional text for screen reader (e.g. "Geospatial Catalog") that can be included in a `<span>` within the heading. 

   This will
   - avoid drift between visible and hidden text, 
   - ensure that all pages get the heading in the same place, 
   - prevent the invisible heading from sliding down in the DOM as new features are added.

- [ ] On the search results page, remove the `<h1 class="sr-only top-content-title">Search</h1>`. It's not necessary and it's better to avoid more than one `<h1>` on a page.

**Comment by cbeer** [comment]
Just confirming  -- the "visible title" is the text "Earthworks" in the header and you want that to be  an `<h1>`, and we should remove  the `<h1 class="sr-only">EarthWorks Geospatial Catalog</h1>` entirely?

**Comment by cbeer** [comment]
> On the search results page, remove the <h1 class="sr-only top-content-title">Search</h1>

For now, would it be sufficient to mark this as `display: none` in the CSS? It's not in a good place for us to override locally and might be a compatibility concern (or, at the very least, we'd probably want to figure out what we were working on in March/April 2018  when we explicitly did what this ticket is asking us to undo.

**Comment by jvine** [comment]
> Just confirming -- the "visible title" is the text "Earthworks" in the header and you want that to be an <h1>, and we should remove the <h1 class="sr-only">EarthWorks Geospatial Catalog</h1> entirely?

Yes

> For now, would it be sufficient to mark this as display: none in the CSS?

Yes
